main.go

```diff
@@ -180,36 +180,23 @@ func main() {
 		fail("Error: %s", err)
 	}
 
+	ticketNumber := ""
 	if len(newCommit.Board) > 0 && newCommit.Board != "NONE" {
-		if newCommit.IsBreakingChange {
-			if len(newCommit.Scope) > 0 {
-				newCommit.Message = fmt.Sprintf("%s(%s)!: <%s> ", newCommit.TicketNumber, newCommit.Scope, newCommit.Type)
-			} else {
-				newCommit.Message = fmt.Sprintf("%s!: <%s> ", newCommit.TicketNumber, newCommit.Type)
-			}
-		} else {
-			if len(newCommit.Scope) > 0 {
-				newCommit.Message = fmt.Sprintf("%s(%s): <%s> ", newCommit.TicketNumber, newCommit.Scope, newCommit.Type)
-			} else {
-				newCommit.Message = fmt.Sprintf("%s: <%s> ", newCommit.TicketNumber, newCommit.Type)
-			}
-		}
-	} else {
-		if newCommit.IsBreakingChange {
-			if len(newCommit.Scope) > 0 {
-				newCommit.Message = fmt.Sprintf("%s(%s)!: ", newCommit.Type, newCommit.Scope)
-			} else {
-				newCommit.Message = fmt.Sprintf("%s!: ", newCommit.Type)
-			}
-		} else {
-			if len(newCommit.Scope) > 0 {
-				newCommit.Message = fmt.Sprintf("%s(%s): ", newCommit.Type, newCommit.Scope)
-			} else {
-				newCommit.Message = fmt.Sprintf("%s: ", newCommit.Type)
-			}
-		}
+		ticketNumber = newCommit.TicketNumber
 	}
 
+	scope := ""
+	if len(newCommit.Scope) > 0 {
+		scope = fmt.Sprintf("(%s)", newCommit.Scope)
+	}
+
+	breakingChange := ""
+	if newCommit.IsBreakingChange {
+		breakingChange = "!"
+	}
+
+	newCommit.Message = fmt.Sprintf("%s%s%s: <%s> ", ticketNumber, scope, breakingChange, newCommit.Type)
```

Comment:
I don't think this accurately captures all of the possible message styles that the previous code did e.g. if the user isn't making use of boards, the angled brackets shouldn't be used